Shawntru/FitLit-refactor
# Feature/post new hydration

## Is this a fix or a feature?
Feature

## What is the change?
the user is now able to post new hydration data to the server 

## Where should the reviewer start?
src/scripts.js line 30

## How should this be tested?
on the main page the dev tools should be open.
Enter an integer in the new hydration input field and press "Enter today's hydration info"
The user's ounces today drank should update to the integer that was used in the input field and
the average ounces should also update. 

## Relevant Ticket(s)
ticket #14 

@Shawntru 
@adavidson02
@sarah01123
